project/plugins.sbt

```diff
@@ -15,5 +15,7 @@ addSbtPlugin("com.typesafe.sbt" % "sbt-coffeescript" % "1.0.0")
 
 addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.3.5")
 
+addSbtPlugin("com.typesafe.sbteclipse" % "sbteclipse-plugin" % "5.1.0")
```

Comment:
yes, run sbt eclipse will automatically download the plugin and generate eclipse project files.